fdbserver/RestoreWorker.actor.cpp

```diff
@@ -80,8 +80,8 @@ ACTOR Future<Void> handlerTerminateWorkerRequest(RestoreSimpleRequest req, Refer
 
 // Assume only 1 role on a restore worker.
 // Future: Multiple roles in a restore worker
-ACTOR Future<Void> handleRecruitRoleRequest(RestoreRecruitRoleRequest req, Reference<RestoreWorkerData> self,
-                                            ActorCollection* actors, Database cx) {
+Future<Void> handleRecruitRoleRequest(RestoreRecruitRoleRequest req, Reference<RestoreWorkerData> self,
```

Comment:
It looks like this is always returning `Void()`. If so, can we change the return type to `void` instead of `Future<Void>`?

Reply:
Ping

Reply:
Fixed. Thanks!